lib/composables/Table.ts

```diff
@@ -78,6 +79,12 @@ export interface TableCellAvatars extends TableCellBase {
   color?: 'neutral' | 'soft' | 'mute'
 }
 
+export interface TableCellComponent extends TableCellBase {
+  type: 'component'
+  component: Component
+  props: Record<string, any>
```

Comment:
I think this should be optional since some component might not require any props 👀

Reply:
Nice 👍